src/js/iqeEnablement.js

```diff
@@ -12,7 +13,11 @@ function init () {
     // must use function here because arrows dont "this" like functions
     window.XMLHttpRequest.prototype.open = function openReplacement(_method, url) { // eslint-disable-line func-names
         this._url = url;
-        return open.apply(this, arguments);
+        req = open.apply(this, arguments);
```

Comment:
This hsould be const assign
```suggestion
        const req = open.apply(this, arguments);
```